perun/utils/__init__.py

```diff
@@ -207,13 +207,14 @@ def run_safely_list_of_commands(cmd_list):
             cprint(err.decode('utf-8'), 'red')
 
 
-def get_stdout_from_external_command(command):
+def get_stdout_from_external_command(command, **kwargs):
     """Runs external command with parameters, checks its output and provides its output.
 
     :param list command: list of arguments for command
+    :param kwargs: additional arguments for the subprocess invocation
     :return: string representation of output of command
     """
-    output = subprocess.check_output([c for c in command if c is not ''], stderr=subprocess.STDOUT)
+    output = subprocess.check_output([c for c in command if c is not ''], stderr=subprocess.STDOUT, **kwargs)
```

Comment:
Is this safe? I have faith in you so I don't want to think in depth about this line of code, but Codacy reported it  (though I ignored it :P).